src/data/init_points.c

```diff
@@ -0,0 +1,81 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   init_points.c                                      :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: reasuke <reasuke@student.42tokyo.jp>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/06/22 15:40:43 by reasuke           #+#    #+#             */
+/*   Updated: 2024/06/22 16:04:27 by reasuke          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "data.h"
+#include "utils.h"
+
+static t_point	**_alloc_points(int row, int col)
+{
+	t_point	**points;
+	int		i;
+
+	points = malloc(sizeof(t_point *) * row);
+	if (points == NULL)
+		error_exit(strerror(errno));
+	i = 0;
+	while (i < row)
+	{
+		points[i] = malloc(sizeof(t_point) * col);
+		if (points[i] == NULL)
+			error_exit(strerror(errno));
+		i++;
+	}
+	return (points);
+}
```

Comment:
This project is not a library. So when an error occurs on allocating memory, there's no problem the program exit at that time because UNIX kernel free its process resources.

Reply:
Thanks. I've understood the good practice to manage allocated memories.
I will consider this in a future issue.